### PR TITLE
Add styles fix for new survey button position in rtl

### PR DESCRIPTION
**Pull request by hariclerry**
This pull request makes the following changes:
-Adds css code to /assests/sass/utils/_helpers.scss

Testing checklist:
- [x] Yellow new survey button was on the right and text was on the left for RTL
- [x] Yellow new survey button is now on the left and text is on the right RTL
- [x] I certify that I ran my checklist

Fixes: https://github.com/ushahidi/platform/issues/3641

#### Screenshot of UI for LTR(English)
Screenshot 2020-10-23 at 15 10 45

#### Screenshot of UI for RTL(Arabic)

Screenshot 2020-10-23 at 15 11 47

### CODE REVIEW

**Comment by hariclerry** [comment]
Hello @rowasc, can you please this pull request? Thank you.

File: assets/sass/utils/_helpers.scss [hunk 1]
```diff
@@ -38,6 +38,10 @@ $LAYOUT
     float: right;
 }
 
+.rtl-button-fix {
```
Comment (Angamanga) [inline assets/sass/utils/_helpers.scss:41]: Sorry I am late on this! We don't need this extra class, we can make use of the same methodology we do in other places in the platform-client. You can read about it here: http://preview.ushahidi.com/platform-pattern-library/master/assets/html/front-end-guidelines/read-direction.html. 

So what we need to do is figure out the right and left positioning of the button in this piece of code: https://github.com/ushahidi/platform-pattern-library/blob/master/assets/sass/5_layouts/_layout-d.scss#L42.

You can see how we handle this in the map-view here: https://github.com/ushahidi/platform-pattern-library/blob/master/assets/sass/4_blocks/_toolbar.scss#L27

I hope this gives you an idea of how to solve it. However well done figuring it out the way you did, this code-base is large and quite complex so its not easy :) Let me know if you need more guidance!

**Review by Angamanga — CHANGES_REQUESTED** [review]
See comments in the code :)

**Comment by hariclerry** [comment]
Hello @Angamanga, thanks for the feedback and guide. However, do I need to make the changes in the file you pointed out: https://github.com/ushahidi/platform-pattern-library/blob/master/assets/sass/5_layouts/_layout-d.scss#L42? I thought it would be this file:  /assests/sass/utils/_helpers.scss, since it is the .right class which is responsible for positioning buttons. I made some changes, please take a look and advice accordingly. Thanks

**Pull request merged**